lte/cloud/go/services/lte/servicers/builder_servicer_test.go

```diff
@@ -163,6 +163,7 @@ func TestBuilder_Build(t *testing.T) {
 			SgiManagementIfaceVlan: "",
 			HeConfig:               &lte_mconfig.PipelineD_HEConfig{},
 			LiUes:                  &lte_mconfig.PipelineD_LiUes{},
+			FeatureSet_5G:          &lte_mconfig.PipelineD_FeatureSet5G{},
```

Comment:
please get used to at least create or modify one test case to verify your code

In one of those test (probably the most general), set feature to true and assert the expected result